lib/generators/suspenders/factory_bot_generator.rb

```diff
@@ -0,0 +1,70 @@
+module Suspenders
+  module Generators
+    class FactoryBotGenerator < Rails::Generators::Base
+      source_root File.expand_path("../../templates/factory_bot", __FILE__)
+
+      def add_factory_bot
+        gem_group :development, :test do
+          gem "factory_bot_rails"
+        end
+
+        Bundler.with_unbundled_env { run "bundle install" }
+      end
+
+      def set_up_factory_bot
+        if default_test_helper_present?
+          insert_into_file Rails.root.join("test/test_helper.rb"), after: "class TestCase" do
+            "\n    include FactoryBot::Syntax::Methods"
+          end
+        elsif rspec_test_helper_present?
+          copy_file "factory_bot_rspec.rb", "spec/support/factory_bot.rb"
+          insert_into_file Rails.root.join("spec/rails_helper.rb") do
+            %(Dir[Rails.root.join("spec/support/**/*.rb")].sort.each { |file| require file })
+          end
+        end
+      end
+
+      def generate_empty_factories_file
+        if default_test_suite?
+          copy_file "factories.rb", "test/factories.rb"
+        elsif rspec_test_suite?
+          copy_file "factories.rb", "spec/factories.rb"
+        end
+      end
+
+      def provide_dev_prime_task
+        copy_file "dev.rake", "lib/tasks/dev.rake"
+      end
+
+      def remove_fixture_definitions
+        if default_test_helper_present?
+          comment_lines "test/test_helper.rb", /fixtures :all/
+        end
+      end
+
+      def create_linting_test
```

Comment:
Should we also lint factories when using RSpec?

Reply:
Yes! Added here: 12b367f0af45fa20a1c7fc1c2a8ecc250b040214